LiteCore/Storage/BothKeyStore.cc

```diff
@@ -149,11 +153,13 @@ namespace litecore {
             }
 
             // Compare the enumerators' keys or sequences:
+            bool oneLeft = true;
             if (_liveImpl && _deadImpl) {
                 if (_bySequence)
                     _cmp = compare(_liveImpl->sequence(), _deadImpl->sequence());
                 else
                     _cmp = _liveImpl->key().compare(_deadImpl->key());
+                oneLeft = false;
```

Comment:
Why not just do the descending check here?  It's cleaner than making a new variable just to test against again below.

Reply:
That's smart, but it reads a little bit indirect. I'll do it.